src/firmware_new/tools/cubemx_regen.sh

```diff
@@ -59,8 +59,8 @@ CUBE_SCRIPT='''
 # Load the project-specific config file (.ioc)
 config load %s
 
-# Generate the peripheral initializations in main.c
-project couplefilesbyip 0
+# Generate the peripheral initializations in separated files
+project couplefilesbyip 1
```

Comment:
this is the change I made to gen into diff files

Reply:
:100: